### ISSUE TITLE
Full Thirst on Death

### ISSUE THREAD

**Issue by rckymtnrfc** [Terrails/StatsKeeper#6]
I've got both Keep Hydration and Keep Thirst set to true but when I die, I have a full Thirst bar.

Stats Keeper-2.1.1
TerraCore-2.0.0
ToughAsNails-1.12-3.0.0.60-universal

Minimal Thirst Amount=8 and Minimal Hydration Amount=0

**Comment by Terrails** [comment]
Weird... I tried it in my workspace and out of it, everything worked, I died and respawned with the same amount of thirst, could you try it once more? Get your thirst down and kill yourself, you should respawn with the same amount

**Comment by rckymtnrfc** [comment]
I'm certain my thirst reset to full, as I was testing your mod.  I'll do some further tests, remove the other mods and see if the problem persists.  Maybe I've got a conflict somewhere.  Thanks

**Comment by Terrails** [comment]
Any update on this?

**Comment by rckymtnrfc** [comment]
Yes, I'd like to report I'm an idiot.

I was doing all this testing in my superflat world and then I noticed I wasn't losing any hunger or water.  My meters always stayed at 100%.  I then did a bunch of testing, pulling out mods, removing all mods, etc....for about 30 minutes.

I just realized it is because I changed the world to Peaceful since this was a test world.

I just added all my mobs back in, fired up a survival world and when I died, my hunger and water meters stayed the same as they were before I died.

Sorry for the trouble.  If you'd like me to never use your mod again, I'll understand.  =)
